### PR TITLE
[dev] refactor utils module

### PR DESCRIPTION
**Pull request by m-kus**
- [x] issue #91 
- [x] follows contribution [guide](https://github.com/keep-starknet-strange/raito/blob/main/CONTRIBUTING.md)
- [x] code change includes tests

### CODE REVIEW

**Review by TAdev0 — APPROVED** [review]
lgtm!

File: src/lib.cairo [hunk 1]
```diff
@@ -4,7 +4,4 @@ pub mod state;
 pub mod codec;
```
Comment (TAdev0) [inline src/lib.cairo:5]: no more merkle_tree.cairo in the lib.cairo file?

Comment (TAdev0) [reply src/lib.cairo:5]: oh yeah you made it a helper. got it

Comment (m-kus) [reply src/lib.cairo:5]: yup

**Review by lana-shanghai — APPROVED** [review]
ДПЕЬ!

**Pull request merged**